src/PackageGraph.js

```diff
@@ -9,6 +9,7 @@ export class PackageGraphNode {
   constructor(pkg) {
     this.package = pkg;
     this.dependencies = [];
+    this.localDependents = [];
```

Comment:
Hah, whoops, I forgot 2.x is still using Arrays, and has no concept of `localDependents` or `localDependencies`...

Reply:
haha yeah that was a disappointing discovery